tests/foreman/api/test_provisioning.py

```diff
@@ -276,3 +277,145 @@ def test_rhel_ipxe_provisioning(
     # assert that the host is subscribed and consumes
     # subsctiption provided by the activation key
     assert provisioning_host.subscribed, 'Host is not subscribed'
+
+
+@pytest.mark.parametrize('pxe_loader', ['bios', 'uefi'], indirect=True)
+@pytest.mark.on_premises_provisioning
+@pytest.mark.rhel_ver_match('[^6]')
+def test_rhel_pxe_provisioning_fips_enabled(
+    request,
+    module_provisioning_sat,
+    module_sca_manifest_org,
+    module_location,
+    provisioning_host,
+    pxe_loader,
+    module_provisioning_rhel_content,
+    provisioning_hostgroup,
+    module_lce_library,
+    module_default_org_view,
+):
+    """Provision a host with host param fips_enabled set to true
+
+    :id: 9e016e1d-757a-48e7-9159-131bb65dc4ef
+
+    :steps:
+        1. Configure satellite for provisioning
```

Comment:
Are we going to `configure satellite for provisioning` in test module with some assertions?
I think fixture `module_provisioning_sat` is use here to get satellite

Reply:
Okay, then this step can be moved into setup section
example
```
:setup:
    1. Configure satellite for provisioning

:steps:
    1. step first
    3. step second
    4. ...
```